etl/src/conversions/event.rs

```diff
@@ -283,7 +284,20 @@ fn convert_tuple_to_row(
                 }
             }
             protocol::TupleData::UnchangedToast => {
-                TextFormatConverter::default_value(&column_schema.typ)?
+                // For unchanged toast values we try to use the value from the old row if it is present
+                // but only if it is not null. In all other cases we send the default value for
+                // consistency. As a bit of a practical hack we take the value out of the old row and
+                // move a null value in its place to avoid a clone because toast values tend to be large.
+                if let Some(row) = old_table_row {
+                    let old_row_value = std::mem::replace(&mut row.values[i], Cell::Null);
+                    if old_row_value == Cell::Null {
+                        TextFormatConverter::default_value(&column_schema.typ)?
+                    } else {
+                        old_row_value
+                    }
+                } else {
+                    TextFormatConverter::default_value(&column_schema.typ)?
```

Comment:
Should we maybe set a marker value so that the consumer knows if this value is actually "skipped"?